<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WindeployQt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/ico/start-here.svg</normaloff>:/ico/start-here.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="Splash">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(68, 68, 68);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="heading">
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>WindeployQt</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="infoText">
          <property name="font">
           <font>
            <family>Inter Semi Bold</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>WindeployQt a front end for windeployQt console application ship with Qt installation.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="continueBtn">
            <property name="font">
             <font>
              <family>Inter Semi Bold</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
width: 94px;
height: 32px;
border: 1px solid &quot;#d6d6d6&quot;;
border-radius: 4px;
background: &quot;#d6d6d6&quot;;
color: &quot;#444&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

QPushButton:hover
{
width: 94px;
height: 32px;
border: 1px solid &quot;#444&quot;;
border-radius: 4px;
background: &quot;#444&quot;;
color: &quot;#fff&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

</string>
            </property>
            <property name="text">
             <string>Continue</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Settings">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="secondHeading">
          <property name="styleSheet">
           <string notr="true">color: &quot;#444&quot;;
font-family: &quot;Inter Semi Bold&quot;;
font-size: 24px;</string>
          </property>
          <property name="text">
           <string>First thing to do</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="headingInfo">
          <property name="styleSheet">
           <string notr="true">color: &quot;#444&quot;;
font-family: &quot;Inter Semi Bold&quot;;
font-size: 14px;</string>
          </property>
          <property name="text">
           <string>Browse the WindeployQt console app location</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="windeploySource">
            <property name="styleSheet">
             <string notr="true">height: 32px;
border: 1px solid &quot;#eee&quot;;
border-radius: 4px;
background: &quot;#eee&quot;;
padding-left: 5px;
padding-right: 5px;
color: &quot;#444&quot;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
width: 94px;
height: 32px;
border: 1px solid &quot;#d6d6d6&quot;;
border-radius: 4px;
background: &quot;#d6d6d6&quot;;
color: &quot;#444&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

QPushButton:hover
{
width: 94px;
height: 32px;
border: 1px solid &quot;#444&quot;;
border-radius: 4px;
background: &quot;#444&quot;;
color: &quot;#fff&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

</string>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="attensionText">
          <property name="styleSheet">
           <string notr="true">color: &quot;#444&quot;;
font-family: &quot;Inter Semi Bold&quot;;
font-size: 12px;</string>
          </property>
          <property name="text">
           <string>*You can also change these settings in future.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="continueBtn2">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
width: 94px;
height: 32px;
border: 1px solid &quot;#d6d6d6&quot;;
border-radius: 4px;
background: &quot;#d6d6d6&quot;;
color: &quot;#444&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

QPushButton:hover
{
width: 94px;
height: 32px;
border: 1px solid &quot;#444&quot;;
border-radius: 4px;
background: &quot;#444&quot;;
color: &quot;#fff&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

</string>
            </property>
            <property name="text">
             <string>Continue</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Home">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="homeHeading">
          <property name="styleSheet">
           <string notr="true">color: &quot;#444&quot;;
font-family: &quot;Inter Semi Bold&quot;;
font-size: 24px;</string>
          </property>
          <property name="text">
           <string>WindeployQt</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="projectSource">
            <property name="styleSheet">
             <string notr="true">height: 32px;
border: 1px solid &quot;#eee&quot;;
border-radius: 4px;
background: &quot;#eee&quot;;
padding-left: 5px;
padding-right: 5px;
color: &quot;#444&quot;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="projectBrowseBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
width: 94px;
height: 32px;
border: 1px solid &quot;#d6d6d6&quot;;
border-radius: 4px;
background: &quot;#d6d6d6&quot;;
color: &quot;#444&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

QPushButton:hover
{
width: 94px;
height: 32px;
border: 1px solid &quot;#444&quot;;
border-radius: 4px;
background: &quot;#444&quot;;
color: &quot;#fff&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

</string>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="qmlDebug">
            <property name="styleSheet">
             <string notr="true">color: &quot;#444&quot;;
font-family: &quot;Inter Semi Bold&quot;;
font-size: 12px;</string>
            </property>
            <property name="text">
             <string>Qml Debug</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="qmlRelease">
            <property name="styleSheet">
             <string notr="true">color: &quot;#444&quot;;
font-family: &quot;Inter Semi Bold&quot;;
font-size: 12px;</string>
            </property>
            <property name="text">
             <string>Qml Release</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="generateBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
width: 94px;
height: 32px;
border: 1px solid &quot;#d6d6d6&quot;;
border-radius: 4px;
background: &quot;#d6d6d6&quot;;
color: &quot;#444&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

QPushButton:hover
{
width: 94px;
height: 32px;
border: 1px solid &quot;#444&quot;;
border-radius: 4px;
background: &quot;#444&quot;;
color: &quot;#fff&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

</string>
            </property>
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="generateQmlBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
width: 110px;
height: 32px;
border: 1px solid &quot;#d6d6d6&quot;;
border-radius: 4px;
background: &quot;#d6d6d6&quot;;
color: &quot;#444&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

QPushButton:hover
{
width: 110px;
height: 32px;
border: 1px solid &quot;#444&quot;;
border-radius: 4px;
background: &quot;#444&quot;;
color: &quot;#fff&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;
}

</string>
            </property>
            <property name="text">
             <string>Generate Qml</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="errorString">
          <property name="styleSheet">
           <string notr="true">font-family: &quot;Inter Semi Bold&quot;;
font-size: 14px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="aboutApp">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Inter Semi Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;close&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Close&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color: &quot;#444&quot;;
font-size: 24px;
font-family: &quot;Inter Semi Bold&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;WindeployQt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: &quot;#bbb&quot;;
font-size: 14px;
font-family: &quot;Inter Semi Bold&quot;;</string>
          </property>
          <property name="text">
           <string>Version 0.1~Beta</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: &quot;#444&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;</string>
          </property>
          <property name="text">
           <string>Deploy Qt projects on windows platforms (XP, Win7 and Win10)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: &quot;#444&quot;;
font-size: 16px;
font-family: &quot;Inter Semi Bold&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#444444;&quot;&gt;Code with&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;/&gt;&lt;span style=&quot; font-size:12pt; color:#ff0105;&quot;&gt;❤️&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;/&gt;&lt;span style=&quot; font-size:12pt; color:#444444;&quot;&gt;by Muhammad Naveed.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: &quot;#444&quot;;
font-size: 14px;
font-family: &quot;Inter Semi Bold&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/muhammadnaveedcis/windeployqt&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;@muhammadnaveed&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBrowse"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMinimize"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_app"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionBrowse">
   <property name="text">
    <string>Browse</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionAbout_app">
   <property name="text">
    <string>About App</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
